CONTRIBUTING.md

```diff
@@ -191,6 +191,11 @@ Translation of pages can be done by simply creating the corresponding page withi
 > [!TIP]
 > When fixing errors in an existing translation, it is suggested to update the page to match the latest version of the English page.
 
+> [!IMPORTANT]
+> Only translate or update languages you can confidently read and proofread.
+> Avoid machine-generated or bulk edits across languages you do not know.
+> When you change an English page, it is fine to leave other languages untouched; native speakers and maintainers will sync them later via the translation dashboards.
```

Comment:
```suggestion
> When you change an English page, it is fine to leave other languages untouched; native speakers and maintainers will sync them later by referring to the [translation dashboards](https://github.com/tldr-pages/tldr-maintenance/issues/127).
```

Edit. Noticed the PR got merged, my GitHub mobile app didn't update the status of it before I made this comment, will apply this change in a new PR separately.